ports/opentyrian2000/Open Tyrian 2000.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIRNAME=opentyrian2000
+GAMEDATADIRNAME=tyrian2000
+GAMEDIR="/$directory/ports/$GAMEDIRNAME"
+CONFDIR="$GAMEDIR/conf/"
+BINARY=opentyrian2000
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+mkdir -p "$GAMEDIR/conf"
+
+export LD_LIBRARY_PATH="$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+export XDG_DATA_HOME="$CONFDIR"
+export TEXTINPUTINTERACTIVE="Y" 
+
+$ESUDO rm -rf ~/.$GAMEDIRNAME
+ln -sfv $GAMEDIR/conf/.$GAMEDIRNAME ~/
```

Comment:
```suggestion
bind_directories ~/.$GAMEDIRNAME $GAMEDIR/conf/.$GAMEDIRNAME
```

Reply:
`$GAMEDIR/conf/.$GAMEDIRNAME` must exist before calling `bind_directories` so in line 29 we want to do `mkdir -p "$GAMEDIR/conf/.$GAMEDIRNAME"`